hamster_gtk/overview/dialogs.py

```diff
@@ -60,8 +62,6 @@ def __init__(self, parent, app, *args, **kwargs):
         self.main_box = self.get_content_area()
         self._daterange = self._get_default_daterange()
 
-        self._charts = False
```

Comment:
Oh, that is just me not being completely familiar with Python. I just needed for `_charts` to be set before calling refresh (i.e. before `_daterange` property is assigned).